src/components/Header/Header.scss

```diff
@@ -1,9 +1,150 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  > * {
+    @include for-desktop-up {
+      padding-left: 30px;
+      padding-right: 30px;
+      max-height: 176px;
+    }
+  }
+
+  a {
+    color: white;
+    text-decoration: underline;
+    text-decoration-color: $color-highlight;
+  }
+}
+
+.govContainer {
+  display: flex;
+  height: 48px;
+  padding-left: 14px;
+  flex-flow: column wrap;
```

Comment:
Minor: I think this CSS could be simplified quite a lot by having one more nesting level of `<div>` to arrange the header items into left/right. (IMO more complex HTML is better than more complex CSS)

Then it's a simple two-column setup (also this is how the ICT header is done). Each column doesn't even need to be flex since it just contains rows.

It'll remove a lot of fiddly things like `flex: 0 0 100%` and `align-content: flex-start;` and having a the funky column-wrap thing.

Reply:
I ended up doing this to satisfy IE.